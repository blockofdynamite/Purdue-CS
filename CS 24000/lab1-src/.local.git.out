[master ae98478] 0 of 100
 6 files changed, 34 insertions(+), 4 deletions(-)
 create mode 100644 .local.git.out
 create mode 100644 .local.total.out
 create mode 100644 total.out
