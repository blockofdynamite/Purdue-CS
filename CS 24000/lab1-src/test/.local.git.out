[master ccee180] 0 of 100
 Committer: CS24000 Account <cs240@data.cs.purdue.edu>
Your name and email address were configured automatically based
on your username and hostname. Please check that they are accurate.
You can suppress this message by setting them explicitly:

    git config --global user.name "Your Name"
    git config --global user.email you@example.com

After doing this, you may fix the identity used for this commit with:

    git commit --amend --reset-author

 3 files changed, 18 insertions(+)
 create mode 100644 test/.local.git.out
 create mode 100644 test/.local.total.out
 create mode 100644 test/total.out
[master 22793f6] 20 of 100
 12 files changed, 79 insertions(+), 12 deletions(-)
 create mode 100755 a.out
 create mode 100644 add.o
 create mode 100755 exp
 create mode 100644 exp_short.o
 create mode 100755 hello
 create mode 100644 hello.c
 create mode 100755 high-low
 rewrite high-low.c (82%)
 create mode 100755 high_low
[master 0c45305] 40 of 100
 3 files changed, 16 insertions(+), 2 deletions(-)
[master 8cc82df] 40 of 100
 4 files changed, 12 insertions(+), 1 deletion(-)
 create mode 100755 test/high-low
[master f6d9bd7] 80 of 100
 6 files changed, 19 insertions(+), 8 deletions(-)
[master a19de9a] 20 of 100
 5 files changed, 25 insertions(+), 9 deletions(-)
[master ed8c301] 20 of 100
 4 files changed, 12 insertions(+), 3 deletions(-)
[master 9aede40] 20 of 100
 4 files changed, 19 insertions(+), 8 deletions(-)
[master 5e20c6c] 20 of 100
 3 files changed, 12 insertions(+), 1 deletion(-)
[master 1e49230] 20 of 100
 3 files changed, 13 insertions(+), 2 deletions(-)
[master afcfe7a] 80 of 100
 5 files changed, 20 insertions(+), 9 deletions(-)
[master 44172cd] 100 of 100
 5 files changed, 16 insertions(+), 5 deletions(-)
[master 7916361] 100 of 100
 2 files changed, 11 insertions(+)
[master de98db1] 100 of 100
 2 files changed, 11 insertions(+)
